README.md

```diff
@@ -68,22 +76,35 @@ Plural's server side is written in elixir, and exposes a graphql api. The fronte
 
 
 ### Developing Web
-To begin developing the web app, install npm & yarn, then run:
+To begin developing the web app, install [node](https://nodejs.org/en/download/) & [yarn](https://classic.yarnpkg.com/en/docs/getting-started/), then run:
 
 ```sh
-cd assets && yarn install && cd -
+cd www && yarn install && cd ..
 make web
 ```
 
+In chrome, you may get a warning saying "Your connection is not private". To resolve it, enable the `chrome://flags/#allow-insecure-localhost` flag and restart your browser.
```

Comment:
This is a hardcore solution. What is yours?

Reply:
I manually bypass by typing "this is unsafe" which is another chrome bypass.  We should probably figure out how to make the certs trusted but either works.